From 6080bcfd4aa3f361e2217eec6a7f43fd12455692 Mon Sep 17 00:00:00 2001
From: Ethan Lee <flibitijibibo@gmail.com>
Date: Fri, 15 Nov 2019 10:55:44 -0500
Subject: [PATCH] HACK: Stub GGP functions

---
 loader/generated/vk_loader_extensions.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/loader/generated/vk_loader_extensions.c b/loader/generated/vk_loader_extensions.c
index c7a55cf11..53512395b 100644
--- a/loader/generated/vk_loader_extensions.c
+++ b/loader/generated/vk_loader_extensions.c
@@ -2198,8 +2198,7 @@ VKAPI_ATTR VkResult VKAPI_CALL CreateStreamDescriptorSurfaceGGP(
     const VkStreamDescriptorSurfaceCreateInfoGGP* pCreateInfo,
     const VkAllocationCallbacks*                pAllocator,
     VkSurfaceKHR*                               pSurface) {
-#error("Not implemented. Likely needs to be manually generated!");
-    return disp->CreateStreamDescriptorSurfaceGGP(instance, pCreateInfo, pAllocator, pSurface);
+    return VK_SUCCESS;
 }
 
 VKAPI_ATTR VkResult VKAPI_CALL terminator_CreateStreamDescriptorSurfaceGGP(
@@ -2207,7 +2206,7 @@ VKAPI_ATTR VkResult VKAPI_CALL terminator_CreateStreamDescriptorSurfaceGGP(
     const VkStreamDescriptorSurfaceCreateInfoGGP* pCreateInfo,
     const VkAllocationCallbacks*                pAllocator,
     VkSurfaceKHR*                               pSurface) {
-#error("Not implemented. Likely needs to be manually generated!");
+    return VK_SUCCESS;
 }
 
 #endif // VK_USE_PLATFORM_GGP
-- 
2.21.0

