From f86f3faf5cb2df7b601753515b0e9a3e983ef1c3 Mon Sep 17 00:00:00 2001
From: Ethan Lee <flibitijibibo@gmail.com>
Date: Wed, 20 Nov 2019 00:30:59 -0500
Subject: [PATCH] GN: Only link to delayimp if d3d9 is enabled

---
 BUILD.gn | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/BUILD.gn b/BUILD.gn
index d930babeb..4cfcca942 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -633,8 +633,10 @@ angle_source_set("libANGLE_base") {
   ]
 
   if (is_win) {
+    if (angle_enable_d3d9) {
+      libs += [ "delayimp.lib" ]
+    }
     libs += [
-      "delayimp.lib",
       "gdi32.lib",
       "user32.lib",
     ]
-- 
2.21.0

