From 911ca13f508bdd85514e0f1fb1ae4d0cdd56bf5c Mon Sep 17 00:00:00 2001
From: Ethan Lee <flibitijibibo@gmail.com>
Date: Wed, 20 Nov 2019 01:19:01 -0500
Subject: [PATCH] UWP: Implement GetPathSeparatorForEnvironmentVar

---
 src/common/system_utils_winuwp.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/common/system_utils_winuwp.cpp b/src/common/system_utils_winuwp.cpp
index 72d550999..24bd8d461 100644
--- a/src/common/system_utils_winuwp.cpp
+++ b/src/common/system_utils_winuwp.cpp
@@ -33,6 +33,12 @@ std::string GetEnvironmentVar(const char *variableName)
     return "";
 }
 
+const char *GetPathSeparatorForEnvironmentVar()
+{
+    // Not supported for UWP
+    return "";
+}
+
 const char *GetSharedLibraryExtension()
 {
     return "dll";
-- 
2.21.0

